CMakeLists.txt

```diff
@@ -208,8 +208,8 @@ set(OTELCPP_PROTO_PATH
 if(WIN32)
   if(BUILD_TESTING)
     if(MSVC)
-      # GTest bug: https://github.com/google/googletest/issues/860
-      add_compile_options(/wd4275)
+      # Warning as error: warning STL4036: <ciso646> is removed in C++20
+      add_compile_options(/wd4996)
```

Comment:
Could this option be applied to test code only instead of making it a global option?

Reply:
As discussed in C++ SIG meeting, 2023-07-12, this is a minor item that will be addressed in a separate PR.